### PR TITLE
Add workshop blog post

### PR DESCRIPTION
**Pull request by henryre**

### CODE REVIEW

File: blog/june-2019-workshop.html [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+  The workshop was structured around the three core Snorkel operations: labeling,
+  transforming, and slicing.
+  Labeling functions were covered on the first day, and transformation and slicing
+  functions covered on the second day.
+  We also explored code from our state-of-the-art SuperGLUE submission,
+  demonstrating transformation and slicing on several of the tasks.
+  The first half of each day was divided between into presentations (both high-level and
+  technical or theoretical deep dives for those interested) and Jupyter notebook-based
+  tutorials.
+  We had two hours of open "office hours"-style sessions in the afternoon during which
```
Comment (paroma) [inline blog/june-2019-workshop.html]: Is it worth mentioning some interests around video, causality etc. to give more structure to the open hours.can do without too

File: blog/june-2019-workshop.html [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+  importantly from our end, get feedback on the upcoming v0.9 release of Snorkel.
+  This upcoming release is our most significant upgrade yet,
+  integrating research work from the last two years
+  (<a href="https://arxiv.org/abs/1711.10160">VLDBJ 2019</a>,
+  <a href="https://arxiv.org/abs/1903.05844">ICML 2019</a>,
+  <a href="https://arxiv.org/abs/1812.00417">SIGMOD 2019</a>,
+  <a href="https://arxiv.org/abs/1810.02840">AAAI 2019</a>,
+  <a href="https://ajratner.github.io/assets/papers/deem-metal-prototype.pdf">DEEM 2018</a>,
+  <a href="https://arxiv.org/abs/1709.01643">NeurIPS 2017</a>),
+  new operators for building and managing training data, the full code to reproduce our
```
Comment (bhancock8) [inline blog/june-2019-workshop.html]: "scripts demonstrating how to reproduce our recent state-of-the-art results on the public benchmark SuperGLUE[link]"

File: blog/june-2019-workshop.html [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+      /**
+      *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
+      *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables*/
+      /*
+      var disqus_config = function () {
+      this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
+      this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
+      };
+      */
+      (function () { // DON'T EDIT BELOW THIS LINE
```
Comment (paroma) [inline blog/june-2019-workshop.html]: is Disqus without ads?

File: blog/june-2019-workshop.html [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+  making it easier to work with massive data sets.
+  Spark integration in particular has been a common feature request, and this was echoed
+  by the workshop attendees.
+
+  <h3>General purpose engine for data set creation</h3>
+
+  We redesigned Snorkel in version 0.9 to be a general purpose engine
+  for programmatically building and managing training data sets across different domains
+  and ML problem types.
+  TODO: EXTRA SENTENCE HERE?
```
Comment (bhancock8) [inline blog/june-2019-workshop.html]: I don't think we need this.

File: blog/june-2019-workshop.html [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+
+  Workshop participants were excited to see modern installation support with pip and conda, as well
+  as lots and lots of unit and integration testing.
+  We also completed full API documentation - requested by everyone at the workshop - during a sprint
+  this past week.
+  All of this will make it easier to get started with, receive support, and contribute to Snorkel.
+
+  <br><br>
+
+  Stay tuned for the full Snorkel v0.9 release post, where we’ll share more details and resources!
```
Comment (bhancock8) [inline blog/june-2019-workshop.html]: Can join the Google Group (https://groups.google.com/forum/#!forum/snorkel-ml) to stay tuned.

File: blog/june-2019-workshop.html [hunk 6]
```diff
@@ -0,0 +1,275 @@
...
+
+  <h2 id="workshop-contents">Overview of Workshop Sessions</h2>
+
+  The workshop was structured around the three core Snorkel operations: labeling,
+  transforming, and slicing.
+  Labeling functions were covered on the first day, and transformation and slicing
+  functions covered on the second day.
+  We also explored code from our state-of-the-art SuperGLUE submission,
+  demonstrating transformation and slicing on several of the tasks.
+  The first half of each day was divided between into presentations (both high-level and
```
Comment (bhancock8) [inline blog/june-2019-workshop.html]: between into

File: blog/june-2019-workshop.html [hunk 7]
```diff
@@ -0,0 +1,275 @@
...
+  recent research around Snorkel, discuss applications and extensions, and most
+  importantly from our end, get feedback on the upcoming v0.9 release of Snorkel.
+  This upcoming release is our most significant upgrade yet,
+  integrating research work from the last two years
+  (<a href="https://arxiv.org/abs/1711.10160">VLDBJ 2019</a>,
+  <a href="https://arxiv.org/abs/1903.05844">ICML 2019</a>,
+  <a href="https://arxiv.org/abs/1812.00417">SIGMOD 2019</a>,
+  <a href="https://arxiv.org/abs/1810.02840">AAAI 2019</a>,
+  <a href="https://ajratner.github.io/assets/papers/deem-metal-prototype.pdf">DEEM 2018</a>,
+  <a href="https://arxiv.org/abs/1709.01643">NeurIPS 2017</a>),
```
[the remark this answers is not in the document]
Comment (henryre) [reply blog/june-2019-workshop.html]: @bhancock8 redundant?

File: blog/june-2019-workshop.html [hunk 8]
```diff
@@ -0,0 +1,275 @@
...
+
+  <h2>Snorkel v0.9 Teaser</h2>
+  Snorkel v0.9 integrates our recent research advances and Snorkel-based open source projects into
+  <b>one modern Python library for building and managing training datasets</b>.
+  In this release, we are:
+
+  <ul>
+    <li>Adding two new core data set operators - transformation functions and slicing functions -
+      alongside labeling functions</li>
+    <li>Upgrading the labeling pipeline using the latest frameworks and algorithmic research</li>
```
[the remark this answers is not in the document]
Comment (henryre) [reply blog/june-2019-workshop.html]: This reflects the following structure (see ### Upgraded labeling pipeline)

File: blog/june-2019-workshop.html [hunk 9]
```diff
@@ -0,0 +1,275 @@
...
+
+  Our upcoming release adds two new core operations as first class citizens in addition to
+  programmatic labeling.
+  The first is <it>transforming</it> data points with transformation functions (TFs), our
+  general
+  <a href="https://hazyresearch.github.io/snorkel/blog/tanda.html">framework for data augmentation</a>.
+  The second is
+  <a href="https://dawn.cs.stanford.edu/2019/06/15/superglue/"><it>slicing</it> training data</a>
+  with slicing functions (SFs), in order to demarcate subsets of the training dataset where
+  classification is more critical or difficult.
```
[the remark this answers is not in the document]
Comment (henryre) [reply blog/june-2019-workshop.html]: not exactly sure what to write here, do you have something mind?

File: blog/june-2019-workshop.html (hunk 3 above)
Comment (henryre) [reply blog/june-2019-workshop.html]: truly no idea

File: blog/june-2019-workshop.html [hunk 10]
```diff
@@ -0,0 +1,275 @@
...
+  We also completed full API documentation - requested by everyone at the workshop - during a sprint
+  this past week.
+  All of this will make it easier to get started with, receive support, and contribute to Snorkel.
+
+  <br><br>
+
+  Stay tuned for the full Snorkel v0.9 release post, where we’ll share more details and resources!
+
+
+  <h2 id="workshop-contents">Overview of Workshop Sessions</h2>
```
[the remark this answers is not in the document]
Comment (henryre) [reply blog/june-2019-workshop.html]: this is just a place to link the slides, tutorials, etc.

File: blog/june-2019-workshop.html (hunk 3 above)
Comment (henryre) [reply blog/june-2019-workshop.html]: looks like no from http://htmlpreview.github.io/?https://github.com/HazyResearch/snorkel/blob/workshop-blog/blog/june-2019-workshop.html

File: blog/june-2019-workshop.html (hunk 5 above)
Comment (henryre) [reply blog/june-2019-workshop.html]: Linked at the bottom, probably enough

**Pull request merged**